src/components/Comments/ContainerComment.js

```diff
@@ -126,7 +127,8 @@ export const ReplyComment = ({
   navigateToReplyView,
   findCommentAndUpdate,
   onCommentLongPressed = () => {},
-  updateVote
+  updateVote,
+  feedId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `feedId` as a parameter in this function is fine, but make sure it's being used correctly within the function body or passed to other functions/components where necessary.
```diff
+   feedId
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->